db/migrate/20150405033056_change_state_migration_columns_to_string.rb

```diff
@@ -0,0 +1,12 @@
+class ChangeStateMigrationColumnsToString < ActiveRecord::Migration
+  def change
+    change_table :state_migration_data do |t|
+      t.change  :median_age, :string
+      t.change  :same_house_1_year_ago, :string
+      t.change  :moved_within_the_same_county, :string
+      t.change  :moved_from_different_county_within_same_state, :string
+      t.change  :moved_from_different_state, :string
+      t.change  :moved_from_abroad, :string
+    end
+  end
+end
```

Comment:
Initially, I saved data as integer, but the data is coming as floats, 0.1, 0.3 etc.. and when saved in the data base as integers are rounded to 0, so changed the data type to be strings.... didn't think it was good to save it as floats in the database... not sure the best way to handle this...